backtester/eventhandlers/strategies/rsi/rsi.go

```diff
@@ -175,12 +175,12 @@ func (s *Strategy) SetDefaults() {
 // this will ensure that RSI can be calculated correctly
 // the decision to handle missing data occurs at the strategy level, not all strategies
 // may wish to modify data
-func (s *Strategy) massageMissingData(data []decimal.Decimal, t time.Time) ([]float64, error) {
-	resp := make([]float64, len(data))
+func (s *Strategy) massageMissingData(d []decimal.Decimal, t time.Time) ([]float64, error) {
```

Comment:
🗻 🐒  Out-Of-Scope but just noting
```suggestion
func (s *Strategy) backfillMissingData(d []decimal.Decimal, t time.Time) ([]float64, error) {
```